packages/launcher/lib/types.ts

```diff
@@ -9,7 +9,9 @@ type BrowserName = 'electron' | 'chrome' | 'chromium' | 'firefox' | string
 
 type BrowserChannel = 'stable' | 'canary' | 'beta' | 'dev' | 'nightly' | string
 
-type BrowserFamily = 'chromium' | 'firefox'
+type BrowserFamily = 'chromium' | 'firefox' | 'webkit' | 'playwright'
+
+type BrowserLaunchEngine = 'cypress' | 'playwright'
```

Comment:
```suggestion
type BrowserFamily = 'chromium' | 'firefox' | 'webkit' | 'playwright'

type BrowserLaunchEngine = 'cypress' | 'playwright'
```

why have both a family and a launchEngine for PW?

one thing i worry about is that people check `browser.family` to determine if certain browser behavior is present in their tests, this won't work the same if the family is `playwright` for Playwright Chromium, for example

Reply:
additionally, i don't think we should change the public schema for browsers for an experimental feature, but this new field should be exposed to users if it's added. is there a way to avoid this?

Reply:
Good point, I will think about it. Originally, I had the idea of launch engine as playwright behaves a bit differently so wanted to make the difference. Probably launch engine would work fine and drop then `playwright` from browser family